Source/Particles/SpeciesPhysicalProperties.H

```diff
@@ -20,7 +20,8 @@ enum struct PhysicalSpecies{unspecified=0, electron, positron, photon, hydrogen}
 
 namespace species
 {
-    static PhysicalSpecies from_string(std::string species)
+    AMREX_FORCE_INLINE
+    PhysicalSpecies from_string(std::string species)
```

Comment:
Fixes the warning:
`./Source/Particles/SpeciesPhysicalProperties.H:23:28`: warning: ‘PhysicalSpecies species::from_string(std::__cxx11::string)’ defined but not used [-Wunused-function]
     static PhysicalSpecies from_string(std::string species)